client/src/app/page/modal/modal.component.html

```diff
@@ -28,10 +28,9 @@ <h4 class="modal-title pull-left">CREATE TASK</h4>
                        (filterVal)="getFilterVal($event)"></app-dropdown-filter>
       </div>
       <div class="btn-group assign-grid">
-        <select class="form-control" name="assignTo" formControlName="assignTo" >
-          <option selected hidden>Choose here</option>
-          <option *ngFor="let user of users; trackBy: TrackElement" [value]="user._id" > {{ user.firstName }} {{ user.lastName }} </option>
-        </select>
+        <app-dropdown-filter class="filter-item-container"
+                       [filterItem]="userDropDown" [cssClassName]="'width-100'"
```

Comment:
`[cssClassName]="'width-100'"` - why do you need use cssClassName attr for this case?